views/schedule_week.tmpl

```diff
@@ -28,42 +33,52 @@
 </div>
 
 <!-- schedule body -->
-<div class="container">
-{{with .Schedule}}
-{{if .Table}}
-<table class="table table-responsive">
-  <thead>
-    <tr>
-      <th>Time</th>
-      {{range .Dates}}
-      <th>{{.Weekday}} {{.Format "02"}}</th>
-      {{end}}
-    </tr>
-  </thead>
-  <tbody>
-    {{range .Table}}
-      <tr>
-        <th scope="row">{{.Hour | printf "%02d"}}:{{.Minute | printf "%02d"}}</th>
-        {{range .Cells}}
-          {{if ne .RowSpan 0}}
-          <td rowspan="{{.RowSpan}}" class="schedule-timeslot schedule-blocks-{{.Item.Block}}" title="{{stripHTML .Item.Desc}}">
-            {{if .Item.PageURL}}
-              <a href="{{url .Item.PageURL}}">{{.Item.Name}}</a>
```

Comment:
In removing the link completely, if JS isnt enabled, there would be no way to click through to shows